LookingMenu/Extensions/UITextField+.swift

```diff
@@ -0,0 +1,17 @@
+import UIKit
+extension UITextField {
+    func paddingLeftTextField() {
+        let paddingView = UIView(frame: CGRect(x: 0,
+                                               y: 0,
+                                               width: 10,
+                                               height: frame.size.height))
+        self.leftView = paddingView
+        self.leftViewMode = .always
+    }
+    
+    func colorPlaceHoder(text : String) {
+        attributedPlaceholder = NSAttributedString(
+            string: text,
+            attributes: [NSAttributedString.Key.foregroundColor: UIColor.white])
+    }
```

Comment:
```suggestion
    func customPlaceHoder(text : String, color: UIColor = .white) {
        attributedPlaceholder = NSAttributedString(
            string: text,
            attributes: [NSAttributedString.Key.foregroundColor: color])
    }
```

Mình nghĩ nên đặt tên và sửa lại code thế này cho nó đúng với mục tiêu của hàm và tăng tính đa dụng